tests/compiler/test_plugin_types.py

```diff
@@ -64,3 +67,13 @@ class CustomDict[K, V](Mapping[K, V]):
 
     with pytest.raises(RuntimeException):
         to_dsl_type(CustomDict[str, str])
+
+    # Check that a warning is produced when implicit cast to 'Any'
+    caplog.clear()
+    to_dsl_type(complex)
```

Comment:
what is this `complex` value?

Reply:
It is the complex numeric type https://docs.python.org/3/library/stdtypes.html#typesnumeric.
I needed a python type that has no equivalent in the dsl